nikola/data/themes/bootstrap3-jinja/assets/css/theme.css

```diff
@@ -175,3 +175,24 @@ article.post-micro {
 .entry-summary {
   margin-top: 1em;
 }
+
+/* hat tip bootstrap/html5 boilerplate */
+@media print {
```

Comment:
Garamond, I’ll accept.

Everything else, please send your complaints to [Bootstrap](https://github.com/twbs/bootstrap/blob/master/dist/css/bootstrap.css#195) and [HTML 5 Boilerplate](https://github.com/h5bp/html5-boilerplate/blob/master/src/css/main.css#L207) first.

> Furthermore, printing in pure black, devoid of any complex color mixing, can make printing much faster, because the printer doesn’t have to mix in any red, green, or blue ink and can focus on getting the task done.